src/commands/createNewProject/ProjectCreateStep/CustomProjectCreateStep.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IHostJsonV2 } from '../../../funcConfig/host';
+import { ScriptProjectCreateStep } from './ScriptProjectCreateStep';
+
+export class CustomProjectCreateStep extends ScriptProjectCreateStep {
+    protected async getHostContent(): Promise<IHostJsonV2> {
+        const hostJson: IHostJsonV2 = await super.getHostContent();
+        hostJson.customHandler = {
+            description: {
+                defaultExecutablePath: '',
+                workingDirectory: '',
+                arguments: []
+            }
```

Comment:
Just verifying that `"enableForwardingHttpRequest": false` shouldn't be included here. I noticed that the example from https://github.com/Azure/azure-functions-core-tools/issues/2052#issue-642663971 includes it but looks like it may have been removed during PR review

Reply:
Yeah it was taken out in PR. Sounds like it already defaults to false if not specified - they were just discussing if it should be there for the sake of discoverability